api-reference/v1.0/resources/groups_overview.md

```diff
@@ -0,0 +1,273 @@
+    "theme": null,
+    "visibility": "Public"
+}
+```
+## Security group and mail-enabled security group
+Security groups are for controlling user access to resources. By checking whether or not a user is a member of a security group, your app can make authorization decisions when that user is trying to access some secure resources in your app. Mail-enabled security groups are the same on principle, but with a group mailbox provisioned by default so the group can receive mail.
+
+**OPENQ** Can SGs be nested?
+
+### Example of security group
+
+```http
+{
+    "@odata.type": "#microsoft.graph.group",
+    "id": "f87faa71-57a8-4c14-91f0-517f54645106",
+    "deletedDateTime": null,
+    "classification": null,
+    "createdDateTime": "2016-07-20T09:21:23Z",
+    "description": "This group is a Security Group",
+    "displayName": "RCG-Read Write-CG-11366",
+    "groupTypes": [],
+    "mail": null,
+    "mailEnabled": false,
+    "mailNickname": "mysg",
+    "preferredLanguage": null,
+    "proxyAddresses": [],
+    "renewedDateTime": "2016-07-20T09:21:23Z",
+    "securityEnabled": true,
+    "theme": null,
+    "visibility": null
+}
+```
+## Dynamic membership 
+All types of groups can have dynamic membership rules which automatically adds or removes users or **devices???** from the group based on criteria over their properties. For example, a "Marketing employees" group should include every user with the department property set to "Marketing", so that new marketing employees are automatically added to the group and employees who leave are automatically removed from the group. This rule can be specified in a "membershipRule" field during group creation as ```"membershipRule": 'user.department -eq "Marketing"'``` GroupType must also include ```"DynamicMembership"```. The following request creates a new Office365 group for the marketing employees: 
+
+```http
+https://graph.microsoft.com/beta/groups
+{
+    "description": "Marketing department folks",
+    "displayName": "Marketing department",
+    "groupTypes": [
+        "Unified",
+        "DynamicMembership"
+    ],
+    "mailEnabled": true,
+    "mailNickname": "marketing",
+    "securityEnabled": false,
+    "membershipRule": 'user.department -eq "Marketing"',
+    "membershipRuleProcessingState": "on"
+}
+```
+
+Learn more about formulating membershipRules in [advanced rules](https://docs.microsoft.com/en-us/azure/active-directory/active-directory-groups-dynamic-membership-azure-portal).
+
+**NOTE** Dynamic membership rules requires the tenant to have a license at tier [Azure Active Directory Premium P1](https://azure.microsoft.com/en-us/pricing/details/active-directory/) or greater.
+
+## Administrator master switch
+Where can an admin disable groups master switch. Planner today will show error. 
+Group creation should succeed if master switch is on.
+
+# END OF ARTICLE, the rest are notes and scribbles
+
+## Distribution group (not sure if this is possible through MS Graph, talk to Dhana)

```

Comment:
We should be pushing folks to using Office Groups.  DGs should really be "legacy".  At some point we should switch off DG creation IMHO.  You should look for messaging guidance from Mike and Kairaz.  They really should write this section.